dotcom-rendering/src/components/Discussion/CommentContainer.tsx

```diff
@@ -194,36 +190,30 @@ export const CommentContainer = ({
 								</li>
 							))}
 						</ul>
-						{!expanded &&
-							comment.metaData &&
-							!!comment.metaData.responseCount &&
-							comment.metaData.responseCount > 3 && (
-								<div
-									css={[
-										topBorder,
-										css`
-											padding-top: ${space[3]}px;
-											padding-bottom: ${space[3]}px;
-										`,
-									]}
+						{!expanded && (
```

Comment:
`expanded` now contains all the necessary information for whether to show a button because it is derived from the actual number of responses shown and the total number that can be shown, no longer hard-coded to a magic number.